src/iplugingame.h

```diff
@@ -65,6 +65,9 @@ class IPluginGame : public QObject, public IPlugin
   // Game plugin should not have requirements:
   std::vector<std::shared_ptr<const IPluginRequirement>> requirements() const final override { return {}; }
 
+  // Game plugin can not be disabled
+  bool enabledByDefault() const final override { return true; }

```

Comment:
That's "nice" because it will fail to compile if plugins try to override this for game plugins but in practice this is never used and it's more that game plugins cannot be enabled/disabled by users (in practice all game plugins are considered disabled except the current one).